content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -254,22 +254,28 @@ Windows has strict compatibility rules, where the host OS version must match the
 * The existing node problem detector is Linux-only and requires privileged containers. In general, we don't expect this to be used on Windows because privileged containers are not supported
 * Not all features of shared namespaces are supported (see API section for more details)
 
-##### Memory Reservations and Handling
+##### Resource Reservations and Handling
 
+###### Memory Reservations
 Windows does not have an out-of-memory process killer as Linux does. Windows always treats all user-mode memory allocations as virtual, and pagefiles are mandatory. The net effect is that Windows won't reach out of memory conditions the same way Linux does, and processes page to disk instead of being subject to out of memory (OOM) termination. If memory is over-provisioned and all physical memory is exhausted, then paging can slow down performance.
 
 Keeping memory usage within reasonable bounds is possible with a two-step process. First, use the kubelet parameters `--kubelet-reserve` and/or `--system-reserve` to account for memory usage on the node (outside of containers). This reduces [NodeAllocatable](/docs/tasks/administer-cluster/reserve-compute-resources/#node-allocatable)). As you deploy workloads, use resource limits (must set only limits or limits must equal requests) on containers. This also subtracts from NodeAllocatable and prevents the scheduler from adding more pods once a node is full.
 
 A best practice to avoid over-provisioning is to configure the kubelet with a system reserved memory of at least 2GB to account for Windows, Docker, and Kubernetes processes.
 
-The behavior of the flags behave differently as described below:
+###### CPU Reservations
+It is better to configure the kubelet with a system reserved CPU of 500m to account for Windows, Docker and other Kubernetes processes. Please note that the 500m value needs to be scaled based on the number of CPU cores available on the Windows node.
```

Comment:
Should I mention to start with 500m explicitly?

Reply:
I don't think so.  This number will vary with the size of vm and works loads. This should be a percentage but I don't know that we have good numbers to suggest at this time.  We can advise to monitor the usage and update according and we should look at generating tests such as http://node-perf-dash.k8s.io/#/builds that would help identify these numbers

Reply:
I was thinking the same but I saw you giving 500m on aks so thought it was a reasonable start. However I am fine changing it to the paragraph you suggested.